src/UglyToad.PdfPig/Graphics/ContentStreamProcessor.cs

```diff
@@ -87,9 +87,8 @@ public override PageContent Process(int pageNumberCurrent, IReadOnlyList<IGraphi
         }
 
         public override void RenderGlyph(IFont font,
-            IColor strokingColor,
-            IColor nonStrokingColor,
-            TextRenderingMode textRenderingMode,
+            CurrentGraphicsState currentState,
+            ParsingOptions parsingOptions,
```

Comment:
no need to pass `ParsingOptions parsingOptions`, you can just access them through `this.ParsingOptions.ClipPaths`